src/table/body-cell/styles.scss

```diff
@@ -26,11 +26,33 @@ $edit-button-padding-right: calc(
 ); // Cell vertical padding + xxs space that would normally come from the button.
 $success-icon-padding-right: calc(#{$edit-button-padding-right} + #{$icon-width-with-spacing});
 
+.expandable-toggle-wrapper {
+  position: absolute;
+  inset-block: 0;
+  display: flex;
+  align-items: center;
+  margin-inline-start: calc(-1 * (#{awsui.$space-m} + #{awsui.$space-xs}));
+}
+
+@mixin cell-offset($padding) {
+  padding-inline-start: $padding;
+
+  @for $i from 1 through 9 {
+    &.body-cell-expandable-level-#{$i} {
+      padding-inline-start: calc($padding + $i * (#{awsui.$space-m} + #{awsui.$space-xs}));
+    }
+  }
+  &.body-cell-expandable-level-next {
+    padding-inline-start: calc($padding + 9 * (#{awsui.$space-m} + #{awsui.$space-xs}));
+  }
```

Comment:
Users can nest more than 9th level but the padding for those will be the same as 9th? We are not stopping them from nesting this deep but also not supporting the styling, isn't it contradicting?

Reply:
The deep nesting will be restricted with the guidelines. However, throwing an error or not showing the data after certain level is risky because sometimes the nesting can be hard to control/restrict on the consumer end. We still want consumers to do so, but if they do not succeed we prefer the data to remain accessible.